package.json

```diff
@@ -1,29 +1,31 @@
 {
   "name": "anvaad-js",
-  "version": "1.4.4",
+  "version": "1.4.3",
```

Comment:
@irvanjitsingh veerji, are we going back on version number for anvaad js?

Reply:
No, the current version on npm is 1.4.3, and I bumped it in a prior PR to 1.4.4 but didn't publish a release. The yarn publish flow also handles bumping the version automatically so I made this change to stay in sync with what's on npm currently.